client/packages/core/src/Reactor.js

```diff
@@ -38,6 +39,14 @@ const OAUTH_REDIRECT_PARAM = "_instant_oauth_redirect";
 
 const currentUserKey = `currentUser`;
 
+let _wsId = 0;
+function createWebSocket(uri) {
+  const ws = new WebSocket(uri);
+  // @ts-ignore
+  ws.id = _wsId++;
```

Comment:
Might want to prefix this with an underscore in case `Websocket` gets a native id field in the future.